music-journal/package.json

```diff
@@ -2,10 +2,22 @@
   "name": "music-sharing",
   "version": "0.1.0",
   "private": true,
+  "main": "dist/server/index.js",
+  "engines": {
+    "node": "10"
+  },
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start"
+    "dev:client": "next src/client",
+    "dev:server": "babel src/server --out-dir dist/server --source-maps --watch",
+    "dev": "npm run dev:client & npm run dev:server",
```

Comment:
I think this should be &&

Reply:
Actually apparently the only difference is && waits for the first to command to be successful and & just runs anyway